multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -269,8 +279,8 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		}
```

Comment:
Question - when endpoint type is `EndpointIPTypePodIP`, do we still care about Service ClusterIP/ports change?

Reply:
Yes, we export Service as an independent Service type of ResourceExport, the importer will check Service type of ResourceExport and create a new MC Service.

Reply:
Do you mean we export ClusterIP and Ports in the ResourceExport for Services? Why?

Reply:
Yes, we exports both for, ports is required for the ResourceExport for Services to create the new MC Service  eg: "antrea-mc-nginx", but ClusterIP can be removed. I will double check and remove it.

Reply:
Ok. I feel we should remove ClusterIP.

Reply:
Done